landoui/assets_src/js/components/Navbar.js

```diff
@@ -0,0 +1,39 @@
+$.fn.landoNavbar = function(){
```

Comment:
I kept it this way because I thought it would be simpler to view the application as a set of components that initialize themselves instead of having some main.js that starts the whole thing.

If the app were to get bigger the main.js would look like
```js

$(document).ready(function () {
  $('.Component1').landoComponent1(); // All the components have to run even if they are not present on the page, because the frontend doens't know that yet.
  $('.Component2').landoComponent2(); 
  $('.Component3').landoComponent3();
  $('.Component4').landoComponent4();
  $('.Component5').landoComponent5();
  $('.PageA').landoPageA();
  $('.PageB').landoPageB(); // has to run even if we're on page A because the frontend doesn't know that yet.
  $('.PageC').landoPageC();
  $('.PageD').landoPageD();
  $('.PageE').landoPageE();
  // and so on...
});
```

So that's why to me it made more sense to keep them in their own files, it hides the uglyness of what is happening (which can be a double edged sword).

Thoughts?